src/MVTSource.js

```diff
@@ -313,11 +327,34 @@ class MVTSource {
       // If the zoom has changed since the request was made, don't draw the tile
       if (this.map.getZoom() != tileContext.zoom) return;
 
-      const arrayBuffer = await response.arrayBuffer();
-      const vectorTile = new VectorTile((new Pbf((new Uint8Array(arrayBuffer)))));
-      this._drawVectorTile(vectorTile, tileContext);
+      // Get the buffer from the response
```

Comment:
This stuff should definitely be in a try/catch but I think it's a bit too verbose. We should be able to tell from the stack trace which call caused the error unless something really funky is happening. Do you think we could get by with a single try/catch based on the errors you've seen?